.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
It'd be great if we could test on newer versions of Node, too... is 6 the minimum? Would it all compile ok on 16 or even 18?

Reply:
Yea, sounds good. I'll see how high I can go before it starts to break 😄

Reply:
Not very far, it turns out.

Thankfully, we don't actually need 90% of the dependencies listed. I've upgraded and simplified. The commit is on it's way, but I haven't tested yet (I have to build outside of my test project because of [annoying reasons]). I will ping once I'm happy that I haven't broken everything.